src/modals/Validate.tsx

```diff
@@ -128,7 +128,7 @@ function Validate(props: ValidateProps) {
                 ];
                 setSmartWallets(tempSmartWallets);
                 localStorage.setItem(
-                    state.account,
+                    `${state.account}-${state.chainId}`,
```

Comment:
I think we should have function like `getAddressKey` or similar to be reused elsewhere

Reply:
updated